### PR TITLE
Updated all table's to use the bootstrap style.

### PR DESCRIPTION
**Pull request by oshotton**

### CODE REVIEW

File: src/site/views/event/tmpl/future.php [hunk 1]
```diff
@@ -20,16 +20,22 @@
...
-    echo "<td><strong>Quantity</strong></td>\n";
-    echo "<td><strong>Remaining</strong></td>\n";
-    echo "</th>\n";
+?>
 
+<h2>Ticket Sales</h2>
+<table class="table table-hover">
+    <thead>
+        <tr>
+            <th>Ticket Type</th>
```
Comment (addshore) [inline src/site/views/event/tmpl/future.php]: Good catch adding this heading! :)

File: src/site/views/events/tmpl/future.php [hunk 2]
```diff
@@ -15,7 +15,7 @@
 ?>
 
 	<!--</style>-->
-	<script type="text/javascript">
+	<script type="text/javascript" xmlns="http://www.w3.org/1999/html">
```
Comment (addshore) [inline src/site/views/events/tmpl/future.php]: xmlns Attribute is only generally used for html tags.
Also not really needed.
See http://www.w3schools.com/tags/att_html_xmlns.asp

File: src/site/views/membertickets/tmpl/default.php [hunk 3]
```diff
@@ -15,7 +15,7 @@
 ?>
 
 <!--</style>-->
-<script type="text/javascript">
+<script type="text/javascript" xmlns="http://www.w3.org/1999/html">
```
Comment (addshore) [inline src/site/views/membertickets/tmpl/default.php]: As with the other comment, xmlns not really needed.

File: src/site/views/events/tmpl/future.php (hunk 2 above)
Comment (oshotton) [inline src/site/views/events/tmpl/future.php]: I don't remember adding these in so not sure how they got there! 😂

**Pull request merged**